src/DatePicker/index.js

```diff
@@ -35,7 +37,7 @@ class DatePicker extends PureComponent {
      * DatePicker month(s) display is based upon currentDate and will start
      * from this to iterate through the months it has to display
      */
-    currentDate: localDateTime,
+    currentMonth: [localDateTime],
```

Comment:
`currentMonth` is an array and can contains multiple months. When I first read your code, it was not clear for me that this variable could contains 2 months. Maybe you should rename it to `currentMonths` or `displayedMonths`? What do you think ?